setup.py

```diff
@@ -4,7 +4,7 @@
 from setuptools import Extension
 import os
 
-VERSION="1.1"
+VERSION="1.1.1"
```

Comment:
I don't remember, do we change the version number in master prior to a release? If we do I think we should bump it all the way to 2.x.y as that will be the next release and even these changes by themselves would justify going to 2.x.y as the hashes will be incompatible.